pkg/storage/memstore/memstore.go

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
Is this memory store reused from other projects?

Reply:
Firas, Sandra and I just talked - I am going to ask Troy if we can make another repo for just storage, since it looks like we're going to be using the same storage in the edv repo and this one

When couchdb support is ready, it'll go into that new repo as well

Reply:
Yes i reused it from aries-framework-go, agree it would make more sense to put that in separate project